### PR TITLE
Update DifferentiableProgramming.md

### PR DESCRIPTION
**Pull request by Lee4Gitr**
Changed  "Numerical computing in Swift  Swift is an
expressive..." to  "Numerical computing in Swift is an
expressive..."

Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves SR-NNNN.

### CODE REVIEW

**Review by dan-zheng — APPROVED** [review]
Thank you!

**Comment by dan-zheng** [comment]
@swift-ci Please smoke test

File: docs/DifferentiableProgramming.md [hunk 1]
```diff
@@ -238,13 +238,12 @@ Differentiation is the process of computing derivatives. See the
...
-applications. Recent proposals have paved the way for low-level numerical
-computing in Swift: [AdditiveArithmetic][SE-0233], SIMD [[1][SE-0229]]
-[[2][SE-0251]], [generic math functions][SE-0246]. However, high-level numerical
-computing applications, including machine learning and artificial intelligence,
-require more work.
+domains, notably deep learning. Numerical computing in Swift is an expressive,
+high-performance language that is a great fit for numerical applications. Recent
+proposals have paved the way for low-level numerical computing in Swift:
+[AdditiveArithmetic][SE-0233], SIMD [[1][SE-0229]] [[2][SE-0251]], [generic math
+functions][SE-0246]. However, high-level numerical computing applications,
```
Comment (dan-zheng) [inline docs/DifferentiableProgramming.md]: We should mention https://swift.org/blog/numerics too!

Comment (Lee4Gitr) [reply docs/DifferentiableProgramming.md]: Absolutely!

Comment (Lee4Gitr) [reply docs/DifferentiableProgramming.md]: Absolutely!

**Review by Lee4Gitr — COMMENTED** [review]
Removed another duplicate "Swift" in documents.

**Pull request merged**